internal/cli/atlas/integrations/describe_test.go

```diff
@@ -61,10 +58,14 @@ func TestDescribe_Run(t *testing.T) {
 	if err := describeOpts.Run(); err != nil {
 		t.Fatalf("Run() unexpected error: %v", err)
 	}
-	assert.Equal(t, `TYPE    API TOKEN   TEAM       CHANNEL
-SLACK   testToken   testTeam    testChannel 
-`, buf.String())
 	t.Log(buf.String())
+	test.VerifyOutputTemplate(t, describeTemplateDatadogOpsGenie, expected)
```

Comment:
this can be represented as a loop over the templates

Reply:
I thought of having templates defined as array in unit test. Alternatively array can be defined in the command itself (but would not be used)

Reply:
no strong opinion here, it's a bit of a test simplification to even maybe use a `t.Run` to encapsulate the cases